### PR TITLE
Update NeoVM

### PR DESCRIPTION
**Pull request by shargon**
Update NeoVM in order to update the compiler with `POW` and `SQRT`

### CODE REVIEW

**Review by erikzhang — APPROVED** [review]

**Comment by erikzhang** [comment]
Need to update the opcodes prices.

**Comment by shargon** [comment]
> Need to update the opcodes prices.

Please adjust `SQRT`, I think that it should be an expensive opcode

File: src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs [hunk 1]
```diff
@@ -156,6 +156,8 @@ partial class ApplicationEngine
             [OpCode.MUL] = 1 << 3,
             [OpCode.DIV] = 1 << 3,
             [OpCode.MOD] = 1 << 3,
+            [OpCode.POW] = 1 << 3,
+            [OpCode.SQRT] = 1 << 11,
```
Comment (erikzhang) [inline src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:160]: @neo-project/ngd-shanghai Can you test the prices?

**Comment by vncoelho** [comment]
I has a greater cost than other basic operators.

`if (value < 4) return BigInteger.One;` maybe it should be Zero as well, no?

**Comment by Qiao-Jin** [comment]
OpCode.POW:

It seems that for func `BigInteger.Pow()`, a small `value` and a big `exponent` will cost most time period, with the limit that the result should be under certain value (`Size `should be less than Integer.MaxSize). Testing 3^160 (whose size is very near to Integer.MaxSize) with OpCode.POW cost about 66 NOP's time.

OpCode.SQRT:

With input as 3^160, test result shows that OpCode.SQRT costs about 1906 NOP's time.

**Review by erikzhang — APPROVED** [review]

**Pull request merged**